2048-game.js

```diff
@@ -0,0 +1,777 @@
+            }
+        }
+    }
+}
+
+function clearGame() {
+    for (i=0; i < arr.length; i++){
+        arr[i].style.backgroundColor = "";
+        arr[i].innerHTML = '';
+    }
+}
+
+
+const startButton = document.getElementById('start');
+startButton.addEventListener("click", initiateGame);
+document.addEventListener("keydown", (e) => {
+    if (e.key == 't' || e.key == 't'){
+        initiateGame();
+    }
+})
+
+const resetButton = document.getElementById('reset');
+resetButton.addEventListener("click", clearGame);
+/*document.addEventListener("keydown", (e) => {
+    if (e.key == "r" || e.key == "R"){
+        clearGame()
+    }
+});*/
+
+function openSquares(arr) {
+    let available = [];
+    for (i = 0; i < arr.length; i++){
+        if (arr[i].style.backgroundColor == ""){
+            available.push(arr[i]);
+        }
+    }
+    return available;
+}
+
+//Functions to determine if tiles can be moved in each direction
+function canItMoveRight(){
+    let counter = 0;
+    for (i = arr.length-1; i >=0; i--){
+        if (arr[i].style.backgroundColor != ""){
+            if (i == 3 || i == 7 || i == 11 || i == 15){
+                let edges = [3,7,11,15];
+                edges.forEach((edgeNo) => {
+                    if (arr[edgeNo - 3].style.backgroundColor != "" && arr[edgeNo - 3].innerHTML == arr[edgeNo - 2].innerHTML){
+                        counter++;
+                    } 
+                    if (arr[edgeNo - 3].style.backgroundColor != "" && arr[edgeNo - 3].innerHTML == arr[edgeNo - 1].innerHTML && arr[edgeNo - 2].style.backgroundColor == "") {
+                        counter++;
+                    }
+                    if (arr[edgeNo - 3].style.backgroundColor != "" && arr[edgeNo - 3].innerHTML == arr[edgeNo].innerHTML && arr[edgeNo - 2].style.backgroundColor == "" && arr[edgeNo - 1].style.backgroundColor == "") {
+                        counter++;
+                    }
+                    if (arr[edgeNo - 2].style.backgroundColor != "" && arr[edgeNo - 2].innerHTML == arr[edgeNo - 1].innerHTML){
+                        counter++;
+                    } 
+                    if (arr[edgeNo - 2].style.backgroundColor != "" && arr[edgeNo - 2].innerHTML == arr[edgeNo].innerHTML && arr[edgeNo - 1].style.backgroundColor == "") {
+                        counter++;
+                    }
+                    if (arr[edgeNo - 1].style.backgroundColor != "" && arr[edgeNo - 1].innerHTML == arr[edgeNo].innerHTML){
+                        counter++;
+                    } 
+                })
+                    
+            }
+            if(i != 3 && i != 7 && i != 11 && i != 15){
+                if (arr[i+1].style.backgroundColor == "" || arr[i+1].innerHTML == arr[i].innerHTML){
+                        counter++;
+                        }  
+                }
+        }
+    }
+    //console.log(`Right counter: ${counter}`)
+    if (counter != 0) return true;
```

Comment:
you can just `return count !== 0`

You can also return early by replacing any of the `counter++` to `return true`
and `return false;` in the end. This is more efficient so you don't have to go through the rest of loop.

ALSO, always use brackets for multi-line if statements. Some people even don't like inline if statements but I find it alright